integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/u-root/u-root/pkg/sh"
+	webboot "github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+// Mount a vfat volume and kexec the kernel within.
+func main() {
+
+	if err := webboot.MountISO("/dev/sda1", ""); err != nil {
+		log.Printf(err.Error())
+	}
+
+	if err := webboot.MountISO("/dev/sd", "mountDir"); err != nil {
```

Comment:
why is it /dev/sd? Is this only ever running in qemu?
If it works for now that is ok and we can change it later.

Reply:
Here I am just trying to mount a non-existent file .This will return an error since the file is none existent hence the test passes.I will change the name to one that clearly shows what I am doing.